Editor/Window/GameLiftPlugin.cs

```diff
@@ -145,6 +179,32 @@ private void OpenTab(string tabName)
                 }
             });
         }
+        
+        public void RefreshProfiles()
```

Comment:
Consider separating the profile logic into a single-purpose class for managing the profile logic that `GameLiftPlugin` could create an instance of and pass it to all tabs that need it. Then the only change needed for the top-level class is to create a dependency

Reply:
For state logic removal. that has been done along with the Unit tests, I will convert it into a full PR to be reviewed as we then will have a baseline for the tests (and include the changes I needed to make). Profile logic will be separated here